gomanifest/internal/golist.go

```diff
@@ -12,8 +12,8 @@ type GoListCmd struct {
 }
 
 // RunGoList ... Actual function that executes go list command and returns output as string.
-func RunGoList(cwd string) (*GoListCmd, error) {
-	goList := exec.Command("go", "list", "-json", "-deps", "-mod=readonly", "./...")
+func RunGoList(cwd string, goExec string) (*GoListCmd, error) {
```

Comment:
Logically the goExec should go as first arg.
```suggestion
func RunGoList(goExec string, cwd string) (*GoListCmd, error) {
```